### ISSUE TITLE
Developing GiftCardPerks.com

### ISSUE THREAD

**Issue by robgpeak** [robgpeak/CompLinks-xclub#338]
@cgladue I just registered the domain GiftCardPerks.com and I want to build out a site that will allow us to sell gift cards and share the "discount" in the form of Rewards Everywhere points.

Here is the concept and steps of what we are looking to accomplish... 

1. Secure Domain Name (Rob) [Done]

2. Select website template (Rob and Chad)
Here's a few choice:
A. https://www.webfulcreations.com/envato/webful_marketplace/html/index.html#
B. http://outline.demo1.wpdance.com/fashion2/
C. http://ps.flytheme.net/themes/sp_market2/en/

3. Integrate the site with the API from Blackhawk Network (Chad) https://developer.blackhawknetwork.com/hawkmarketplace/docs/ordering-digital-gift-cards-egifts

4. We want to try and follow the flow on the whiteboard. 
 I invited you to be an editor on the whiteboard as well and you should have received a separate email invitation. https://miro.com/app/board/o9J_kxYZxJ0=/

I know it's sometimes hard for us to connect on a screen share session during the normal work day, so I am going to create a loop screen cast video of how I am thinking about it so you can reference it and then we can have a regular phone call if need be to discuss some of the details.
https://www.loom.com/share/0c7277d9c78e470c8c5f567793578281

As a quick follow up video I think it made sense to make another short video talking about the overall process of how Blackhawk Network will be the merchant of record for the credit card transaction when someone buys a gift card on our site. https://www.loom.com/share/c8ea413f1e834f37b0288e034d158ba6

**Comment by cgladue** [comment]
hey rob, can you maybe look at envato for a template you like ?  they have
nice HTML5 Bootstrap templates that would be ideal .. the URL is here:

https://themeforest.net/category/site-templates?sort=sales

skip any that are wordpress or woocommerce type themes look for Bootstrap
and HTML5 keywords

i can come up with a few suggestions too if you would like ?

**Comment by robgpeak** [comment]
Yeah please do look for a theme that you think you could easily work with
to get the site to do what we are looking for.

Thanks,

Rob

**Sent from my phone

**Comment by cgladue** [comment]
@robgpeak what do you think of these look/feel https://themeforest.net/item/couponday-clean-and-premium-coupon-template/7752973

we can adjust images and colors but the basic design

**Comment by robgpeak** [comment]
Hey Chad, that template looks fine, but in the list of priorities, this would be close to the bottom because the company that we were really developing this site for (PCH), back out of the deal.  So I would rather get the SMS feature up and running first and polished.

**Comment by cgladue** [comment]
yea absolutely, just wanted to at least choose a design

**Comment by robgpeak** [comment]
Cool.  I like it, it seems like it would be super easy to work with.  We will definitely revisit it when we have a company that needs it as it's redemption method.
